lib/hanami/cli.rb

```diff
@@ -26,8 +26,11 @@ def run(arguments: ARGV)
         true
       end
 
-      def register(name, command)
-        Hanami::Cli.register(name, command)
+      def register(*args)
```

Comment:
Just a stylistic issue: maybe is better to use two arguments for method definition?
```
def register(*names, command)
  names.each { |name| Hanami::Cli.register(name, command) }
end
```